JohnsonCountyJail.yaml

```diff
@@ -882,428 +886,374 @@ entityDefinitions:
-        column: Person_Post
-    name: "Postedby2"
-  Postedby3:
-    fqn: 'ol.postedby'
-    src: JohnsonCounty_IowaCity_JCBond
-    dst: JohnsonCounty_IowaCity_BondRelative
-    entitySetName: "JohnsonCounty_IowaCity_JCPostedBy"
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatCombineTransform>
+          transforms:
+          - !<transforms.ColumnTransform>
+            column: "Jail_ID"
+          - !<transforms.ValueTransform>
+            value: "2"
+      general.datetime:
+        type: "general.datetime"
+        column: "Arrest_Date"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["M/d/yyyy", "d-MMM-yy", "MM/dd/yyy", "M/dd/yyyy", "MM/d/yyyy", "dd-MMM-yy", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/Chicago"
+    name: "JohnsonCounty_IowaCity_JCChargedWith2"
+
+  JohnsonCounty_IowaCity_JCCommittedBy:
+    fqn: "ol.committedby"
+    entitySetName: "JohnsonCounty_IowaCity_JCCommittedBy"
+    src: "JohnsonCounty_IowaCity_JCInmates"
+    dst: "CommittingAuthority"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Person_Post
-    name: "Postedby3"
-  Kinto:
-    fqn: 'ol.kinto'
-    src: JohnsonCounty_IowaCity_JCInmates
-    dst: JohnsonCounty_IowaCity_BondRelative
-    entitySetName: "JohnsonCounty_IowaCity_JCKinTo"
+        type: "ol.id"
+        column: "Jail_ID"
+      ol.commitdatetime:
+        type: "ol.commitdatetime"
+        column: "Start_Date"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["M/d/yyyy", "d-MMM-yy", "MM/dd/yyy", "M/dd/yyyy", "MM/d/yyyy", "dd-MMM-yy", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/Chicago"
+    name: "JohnsonCounty_IowaCity_JCCommittedBy"
+
+  JChas1:
+    fqn: "ol.has"
+    entitySetName: "JohnsonCounty_IowaCity_JCHas"
+    src: "JohnsonCounty_IowaCity_JCInmates"
+    dst: "physicalappearance"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Person_Post
-    name: "Kinto"
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatCombineTransform>
+          transforms:
+          - !<transforms.ColumnTransform>
+            column: "Jail_ID"
+          - !<transforms.ValueTransform>
+            value: "1"
+    name: "JChas1"
 
-#####################################
-## AGENCIES <- INMATES -> OFFICERS ##
-#####################################
+  JChas2:
+    fqn: "ol.has"
+    entitySetName: "JohnsonCounty_IowaCity_JCHas"
+    src: "JohnsonCounty_IowaCity_JCInmates"
+    dst: "PersondetailsCJ"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatCombineTransform>
+          transforms:
+          - !<transforms.ColumnTransform>
+            column: "Jail_ID"
+          - !<transforms.ValueTransform>
+            value: "2"
+    name: "JChas2"
 
   JohnsonCounty_IowaCity_JCArrestedBy:
```

Comment:
For all of these associations linking the officer and the inmate, it should not link to the general.person officer entity set but the ol.employee entity set - JohnsonCounty_IowaCity_JCOfficerInfo. If you want to make it less syntactically awkward you could just change this entity set name to JohnsonCounty_IowaCity_JCOfficers (and keep the other one, "...OfficerPerson"